content/applications/sales/point_of_sale/shop/serial_numbers.rst

```diff
@@ -0,0 +1,64 @@
+==============================
+Serial numbers and lots in PoS
+==============================
+
+Working with **serial numbers** and **lots** allows tracking your products' movements. When products
+are tracked, the system identifies their location based on their last movement.
+
+To enable traceability, go to :menuselection:`Point of Sale --> Products --> Products`. Then,
+select a product and check the :guilabel:`Tracking By Unique Serial Number` or the
+:guilabel:`Tracking By Lots` box in the :guilabel:`Inventory` tab.
+
+.. image:: serial_numbers/product-form-traceability.png
+   :align: center
+   :alt: Enable traceability settings
+
+Serial numbers and lots import
+==============================
+
+You can import serial numbers in Point of Sale. To do so, select a **sales order** or a
+**quotation** containing tracked products. Then, agree to load the **Lots or Serial Numbers** linked
+to the :abbr:`SO (sales order)`.
+
+.. image:: serial_numbers/importing-sn.png
+   :align: center
+   :width: 480
+   :alt: Pop-up window for serial number import
+
+.. seealso::
+   - :doc:`../shop/sales_order`
+
+The imported tracking numbers appear below the tracked products. You can modify them by clicking on
+the list-view button next to the products.
+
+.. image:: serial_numbers/pos-sn-imported.png
+   :align: center
+   :width: 480
+   :alt: Imported serial numbers in POS interface
+
+Serial numbers and lots creation
+================================
+
+If a tracked product is available in your POS, adding the product to the cart opens a pop-up window
+where you can type the product's serial or lot numbers. To add more than one of the same tracked
+products, click on **enter** to validate and start a new line.
+
+.. tip::
+   To avoid typing the tracking numbers, add a label to your products. This way, you can scan their
+   serial or lots numbers.
```

Comment:
it's kind of a good tip, but I assume that most shops don't manufacture the products they sell directly at the shop. From the PoS point of view, you often download a file with all the existing S/N made by the manufacturers.

Reply:
You got a point, I removed the tip and added "or scan" previously in case they already have the labels printed